include/kframework/serialization.md

```diff
@@ -53,8 +53,8 @@ Address/Hash Helpers
     rule #sender("")  => .Account
     rule #sender(STR) => #addr(#parseHexWord(Keccak256(STR))) requires STR =/=String ""
 
-    syntax Int ::= #addrFromPrivateKey ( String ) [function]
- // --------------------------------------------------------
+    syntax Int ::= #addrFromPrivateKey ( String ) [function, concrete]
+ // ------------------------------------------------------------------
     rule #addrFromPrivateKey ( KEY ) => #addr( #parseHexWord( Keccak256 ( Hex2Raw( ECDSAPubKey( Hex2Raw( KEY ) ) ) ) ) )
 ```
```

Comment:
This isn't the right way to mark it as concrete I don't think. Instead, name the rule `rule [addrFromPrivateKey]: ...`, and add the rule name to `tests/specs/concrete-rules.txt`, and to `KEVM.concrete_rules()` in `kevm.py`.